quarkus.ls.ext/com.redhat.quarkus.ls/pom.xml

```diff
@@ -0,0 +1,175 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>com.redhat.microprofile</groupId>
+	<artifactId>com.redhat.quarkus.ls</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
```

Comment:
<version>0.0.8-SNAPSHOT</version>

Reply:
Changed to use 0.8.0-SNAPSHOT instead to match mp.ls

Reply:
right, that's what I meant ;-)